src/bundle/Resources/public/js/scripts/admin.linkmanager.list.js

```diff
@@ -0,0 +1,7 @@
+(function(doc) {
+    const statusField = doc.getElementById('search_data_status');
```

Comment:
What is the HTML structure of element with id `search_data_status`?

Reply:
`#search_data_status`  is `<select>` (link status filter)